src/apps/material/material.tsx

```diff
@@ -36,38 +37,54 @@ export interface MaterialProps {
   wid: WorkId;
 }
 
+export const getWorkManifestation = (work: WorkMediumFragment) => {
+  return work.manifestations.latest;
+};
+
+export const getManifestationFromType = (
+  type: string,
+  work: WorkMediumFragment
+) => {
+  const allManifestations = orderManifestationsByYear(work.manifestations);
+
+  const allManifestationsThatMatchType = allManifestations.filter(
+    (item) => item.materialTypes[0].specific === type
+  );
+
+  return allManifestationsThatMatchType[0];
+};
+
 const Material: React.FC<MaterialProps> = ({ wid }) => {
   const t = useText();
-  const [typeState, setTypeState] = useState<null | string>(null);
+
+  const [currentManifestationState, setCurrentManifestationState] = useState<
+    ManifestationsSimpleFragment["latest"] | null
+  >(null);
+
   const { data, isLoading } = useGetMaterialQuery({
     wid
   });
 
```

Comment:
I think we should have the loading and error handling up here before the useEffect.
So when we are past all that:
// TODO: handle erro
We can destruct the work object.

Reply:
I get this eslint error if I move it above useEfffect:

React Hook "useEffect" is called conditionally. React Hooks must be called in the exact same order in every component render